File: vapor-app/src/main/java/com/tevinjeffrey/vapor/okcloudapp/model/CloudAppJsonAccount.java
package com.tevinjeffrey.vapor.okcloudapp.model;

public class CloudAppJsonAccount {
    User user;

    class User {
        String email;
        String password;
        String currentPassword;
        String domain;
        String domainHomePage;
        boolean acceptTos;
        boolean privateItems;
    }
}

File: vapor-app/src/main/java/com/tevinjeffrey/vapor/okcloudapp/model/CloudAppJsonAccount.kt
package com.tevinjeffrey.vapor.okcloudapp.model

class CloudAppJsonAccount {
    internal var user: User? = null
    internal inner class User {
        var email: String? = null
        var password: String? = null
        var currentPassword: String? = null
        var domain: String? = null
        var domainHomePage: String? = null
        var acceptTos: Boolean = false
        var privateItems: Boolean = false
    }
}
